sqlx-core/Cargo.toml

```diff
@@ -156,8 +157,8 @@ tokio-stream = { version = "0.1.2", features = ["fs"], optional = true }
 smallvec = "1.4.0"
 url = { version = "2.1.1", default-features = false }
 uuid = { version = "0.8.1", default-features = false, optional = true, features = ["std"] }
-webpki = { version = "0.21.0", optional = true }
-webpki-roots = { version = "0.21.0", optional = true }
+webpki = { version = "0.22.0", optional = true }
```

Comment:
```suggestion
```

Accidentally added this back in when resolving the merge conflict.

Reply:
Fixed